scripts/Marquee.js

```diff
@@ -0,0 +1,47 @@
+const baseUrl = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/`;
+
+class StockData {
```

Comment:
the Marqee class should contain all the logic of fetching the data and rendering it into the DOM. See ITC solution to the project when published.